view.c

```diff
@@ -4,6 +4,7 @@
 #include <ctype.h>
 #include <errno.h>
 #include <limits.h>
+#include <assert.h>
```

Comment:
I have removed them in my last commit. I don't understand your reasoning though, as asserts are useful for debugging and vis is already configured to remove them from release builds. I originally inserted those asserts there because they would have caught a weird segfault I had to debug earlier.

Reply:
gdb can also catch segfaults. The problem with `assert()` is that its
implemented as a macro and can have side effects when used incorrectly.
In general its usage also makes me think the author doesn't understand
the code they have written. There was nothing wrong with your usage but
I don't want to open that can of worms.